common/persistence/nosql/nosqlplugin/cassandra/workflow_utils.go

```diff
@@ -1357,6 +1454,75 @@ func createWorkflowExecution(
 	return nil
 }
 
+func isRequestRowType(rowType int) bool {
```

Comment:
add a specific unit test for this and also leave a comment on top of `rowTypeWorkflowRequestStart` as a reminder to update this when new request types are introduced.